spec/etl/items/importers/quality_metrics_spec.rb

```diff
@@ -56,6 +56,11 @@
       )
     end
 
+    it 'returns without running content_quality_service if #get_content returns nil' do
+      allow_any_instance_of(Dimensions::Item).to receive(:get_content).and_return(nil)
```

Comment:
Instead of stubbing this method, I would rather to create an Item that has no content. This will help in refactorings as we don't mock the implementation.

We could use this:

```ruby
    payload = GovukSchemas::RandomExample.for_schema(notification_schema: "detailed_guide")
    ...
```

We can even modify FactoryGirl to create a dimension item with no content on a explicit way.
What do you think?